release_notes/372-release-notes.adoc

```diff
@@ -8,4 +8,265 @@ toc::[]
+
+[id="fixable-cvss6-disabled"]
+=== Policy for CVEs with fixable CVSS of 6 or greater disabled by default 
+
+Beginning with this release, the `Fixable CVSS >= 6 and Privileged` policy is no longer enabled by default for new {product-title-short} installations. The configuration of this policy is not changed when upgrading an existing system. A new policy `Privileged Containers with Important and Critical Fixable CVEs`, which gives an alert for containers running in privileged mode that have important or critical fixable vulnerabilities, has been added. The new policy is also in the `Deploy` life cycle. Known vulnerabilities make it easier for adversaries to exploit your application, and highly-privileged containers pose greater risk. You can fix these high-severity vulnerabilities by updating to a newer version of the affected components or running your container with lower privileges.
+
+[id="deprecation-notices-372"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed, refer to the table below. Additional information about some removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+- GA: General Availability
+- TP: Technology Preview
+- DEP: Deprecated
+- REM: Removed
+
+.Deprecated and removed features tracker
+[cols="3,1,1,1",options="header"]
+|====
+|Feature | {product-title-short} 3.70 |{product-title-short} 3.71 | {product-title-short} 3.72
+
+|`RenamePolicyCategory` and `DeletePolicyCategory` API endpoints
+|DEP
+|DEP
+|DEP
+
+|Support for violation tags and process tags
+|DEP
+|DEP
+|REM
+
+| Permissions: `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, `User`, `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, `Risk`, `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, `SignatureIntegration`, `ImageComponent`
+| GA
+| DEP
+| DEP
+
+| Retrieving groups by property
+| GA
+| DEP
+| DEP
+
+| `vulns` fields of `storage.Node` object in response payload of `v1/nodes` 
+| GA
+| DEP
+| DEP
+
+| `/v1/cves/suppress` and `/v1/cves/unsuppress`
+| GA
+| DEP
+| DEP
+
+| `ids` field in the `/v1/cves/suppress` and `/v1/cves/unsuppress` API payload 
+| GA
+| DEP
+| DEP
+
+| `cves.ids` field of the `storage.VulnerabilityRequest` object in the response of `VulnerabilityRequestService` endpoints
+| GA
+| DEP
+| DEP
+
+| Scanning support for Ubuntu 21.10
+| GA
+| GA
+| DEP
+
+| Permission `ClusterCVE`
+| GA
+| GA
+| DEP  
+
+| `Label` and `Annotation` search options
```

Comment:
In the actual YAML, `label` and `annotation` are actually lowercase. This could just be _label_ and _annotation_ I think.

Reply:
added "in RHACS" to clarify that these are ACS search options